includes/class-wc-payments-blocks-payment-method.php

```diff
@@ -58,7 +58,7 @@ public function get_payment_method_script_handles() {
 				'wc-blocks-checkout-style',
 				plugins_url( 'dist/blocks-checkout.css', WCPAY_PLUGIN_FILE ),
 				[],
-				'1.0',
+				WC_Payments::get_file_version( 'dist/checkout.css' ),
```

Comment:
Getting the version from the build file, so that it changes at each release and we can effectively cache bust it - otherwise we incur the risk of providing an older build (because it could be cached by the customer's browser or the merchant's CDN)